### PR TITLE
python3Packages.aiopvpc: init at 2.0.2

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module to download Spanish electricity hourly prices (PVPC)

https://github.com/azogue/aiopvpc

This is a Home Assistant dependency
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120266` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.aiopvpc</li>
    <li>python39Packages.aiopvpc</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120266` at a0a064af run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.aiopvpc</li>
    <li>python39Packages.aiopvpc</li>
  </ul>
</details>

File: pkgs/development/python-modules/aiopvpc/default.nix [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    aiohttp
+    pytz
+    async-timeout
+  ];
+
+  checkInputs = [
+    pytest-asyncio
+    pytest-timeout
+    pytestCheckHook
+  ];
```
Comment (mweinelt) [inline pkgs/development/python-modules/aiopvpc/default.nix:42]:
```
=============================== warnings summary ===============================
tests/test_pvpc.py: 11 warnings
  /nix/store/69jp5jhv1v79zk2dlv5vy9ssz89bik9w-python3.8-pytest-6.2.2/lib/python3.8/site-packages/_pytest/python.py:172: PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
    - anyio
    - pytest-asyncio
    - pytest-tornasync
    - pytest-trio
    - pytest-twisted
    warnings.warn(PytestUnhandledCoroutineWarning(msg.format(nodeid)))

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================== 1 passed, 14 skipped, 11 warnings in 0.47s ==================
```

That's a bit disappointing. Can you investigate, why it skips so many tests even with `pytest-asyncio`?

**Review by SuperSandro2000 — APPROVED** [review]
other than mweinelt remark LGTM

File: pkgs/development/python-modules/aiopvpc/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/aiopvpc/default.nix:42]: Some tests require network access. They are skipped. Let me see if we can increase the number of tests running successfully as there are fixtures present in the repo.

Comment (lukegb) [reply pkgs/development/python-modules/aiopvpc/default.nix:42]: Anything left to do here before merging?

**Comment by fabaff** [comment]
No response on azogue/aiopvpc#11.

File: pkgs/development/python-modules/aiopvpc/default.nix (hunk 1 above)
Comment (azogue) [reply pkgs/development/python-modules/aiopvpc/default.nix:42]: Hi there @fabaff and @lukegb, 
the testing environment in `aiopvpc` is using `pytest-aiohttp`, not `pytest-asyncio`. I could try to make the switch, if that's the problem?

Comment (azogue) [reply pkgs/development/python-modules/aiopvpc/default.nix:42]: Just checked that I can change `pytest-aiohttp` for `pytest-asyncio`, add the decorators over async tests (`@pytest.mark.asyncio`), and everything runs as expected, so, no need to change anything outside `aiopvpc`, I'll publish another release with the change :)

Comment (fabaff) [reply pkgs/development/python-modules/aiopvpc/default.nix:42]: Thanks for the feedback, @azogue.

**Pull request merged**